integration-tests/deployment/ccip/propose.go

```diff
@@ -217,3 +217,40 @@ func GenerateAcceptOwnershipProposal(
 		batches,
 		timelock.Schedule, "0s")
 }
+
+func CreateSingleChainMCMSOps(
+	ops []mcms.Operation,
+	selector uint64,
+	description string,
+	tl *owner_helpers.RBACTimelock,
+	mcm *owner_helpers.ManyChainMultiSig,
+) (*timelock.MCMSWithTimelockProposal, error) {
+	chainId := mcms.ChainIdentifier(selector)
```

Comment:
I think a better helper function would be one like:
```func BuildProposalMetadata(state CCIPState, chains []uint64) (map[mcms.ChainIdentifier]common.Address, map[mcms.ChainIdentifier]mcms.ChainMetadata, error)```

Then used inline to build the proposal metadata before a timelock.NewMCMSWithTimelockProposal call

Reply:
let's do it in a follow up to get this test running in CI for now